conductor-modules/glide/src/main/java/com/bluelinelabs/conductor/GlideProvider.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Lalafo.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bluelinelabs.conductor;
+
+import com.bumptech.glide.RequestManager;
+
+public interface GlideProvider<T extends RequestManager> {
```

Comment:
what about placing this classes under `.glide` package?

Reply:
also switch the whole global package to `com.lalafo.conductor`, as this is our own module.